Zend/Optimizer/zend_inference.c

```diff
@@ -3911,9 +3916,36 @@ static zend_always_inline zend_result _zend_update_type_info(
 			UPDATE_SSA_OBJ_TYPE(zend_ce_closure, /* is_instanceof */ false, ssa_op->result_def);
 			break;
 		case ZEND_FETCH_CONSTANT:
-		case ZEND_FETCH_CLASS_CONSTANT:
 			UPDATE_SSA_TYPE(MAY_BE_RC1|MAY_BE_RCN|MAY_BE_ANY|MAY_BE_ARRAY_KEY_ANY|MAY_BE_ARRAY_OF_ANY, ssa_op->result_def);
 			break;
+		case ZEND_FETCH_CLASS_CONSTANT:
+			{
```

Comment:
Please conform to the general styling of the optimizer. Curly on the same line as `:`, `&&`/`||` on the new line, indented by spaces. no blank lines after `if (`, etc.

There's an ongoing discussion about possibly enfording styling, but let's stick with what we have until we actually decide on anything.

Reply:
I hope that formatting is fine now - I tried to find any deviation, but of course it may be possible that there are some omissions